client/my-sites/post-share/index.jsx

```diff
@@ -0,0 +1,490 @@
+					}
+					description={ translate( 'Live chat support and no advertising.' ) }
+					list={ [
+						translate( 'Live chat support' ),
+						translate( 'No Advertising' )
+					] }
+					plan={ PLAN_BUSINESS }
+					title={ translate( 'Upgrade to a Business Plan!' ) }
+				/>
+			);
+		}
+
+		// TODO: get from Redux
+		const scheduledItems = [
+			{
+				service: 'twitter',
+				handle: 'styleandgear',
+				timestamp: 'Tue, Jan 29, 2017 at 5:00 PM',
+				message: 'Do you have a trip coming up? Bla some more text'
+			},
+			{
+				service: 'tumblr',
+				handle: 'tasha',
+				timestamp: 'Tue, Jan 28, 2017 at 3:35 PM',
+				message: 'Do you have a trip coming up? Bla some more text'
+			},
+			{
+				service: 'facebook',
+				handle: 'Style and Gear',
+				timestamp: 'Tue, Jan 28, 2017 at 3:35 PM',
+				message: 'Do you have a trip coming up? Bla some more text'
+			}
+		];
+
+		return scheduledItems.map(
+			( item, index ) => this.renderFooterSectionItem( item, index )
+		);
+	}
+
+	renderPublishedList() {
+		return 'published';
+	}
+
+	renderFooter() {
+		const { footerSection } = this.state;
+
+		return (
+			<div className="post-share__footer">
+				<SectionNav className="post-share__footer-nav" selectedText={ 'some text' }>
+					<NavTabs label="Status" selectedText="Published">
+						<NavItem
+							selected={ footerSection === PostShare.FOOTER_SECTION_SCHEDULED }
+							count={ 4 }
+							onClick={ this.setFooterSection( PostShare.FOOTER_SECTION_SCHEDULED ) }
+						>
+							Scheduled
+						</NavItem>
+						<NavItem
+							selected={ footerSection === PostShare.FOOTER_SECTION_PUBLISHED }
+							count={ 2 }
+							onClick={ this.setFooterSection( PostShare.FOOTER_SECTION_PUBLISHED ) }
+						>
+							Published
+						</NavItem>
+					</NavTabs>
+				</SectionNav>
+				<div className="post-share__scheduled-list">
+					{ footerSection === PostShare.FOOTER_SECTION_SCHEDULED &&
+						this.renderScheduledList()
+					}
+				</div>
+				<div className="post-share__published-list">
+					{ footerSection === PostShare.FOOTER_SECTION_PUBLISHED &&
+						this.renderPublishedList()
+					}
+				</div>
+			</div>
+		);
+	}
+
+	render() {
+		if ( ! this.props.isPublicizeEnabled ) {
+			return null;
+		}
+
+		if ( ! this.props.planHasRepublicizeFeature ) {
+			return ( <div className="post-share">
+				<Banner
+					feature="republicize"
+					title={ this.props.translate( 'Unlock the ability to re-share posts to social media' ) }
+					callToAction={ this.props.translate( 'Upgrade to Premium' ) }
+					description={ this.props.translate( 'Get unlimited premium themes, video uploads, monetize your site and more.' ) }
+				/>
+			</div> );
+		}
+
+		const classes = classNames( 'post-share__wrapper', {
+			'has-connections': this.hasConnections()
+		} );
+
+		return (
+			<div className="post-share">
+				{ this.props.requesting &&
+					<Notice status="is-warning" showDismiss={ false }>
+							{ this.props.translate( 'Sharing…' ) }
```

Comment:
`…` > `...`

The existing string is [here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/posts/post-share.jsx#L149), so if we can change that one before committing this one, it'll be better.